packages/webpack/template-webpack-plugin/src/LynxTemplatePlugin.ts

```diff
@@ -123,6 +123,8 @@ export interface TemplateHooks {
     outputName: string;
     mainThreadAssets: Asset[];
     cssChunks: Asset[];
+    isAsync: boolean;
+    entryNames: string[];
```

Comment:
Adding a new API  should be a minor change, and also update:

https://github.com/lynx-family/lynx-stack/blob/fd233bd970d1f64e24e7e809b65c43654a1c7577/packages/webpack/css-extract-webpack-plugin/package.json#L62